futurenhs.app/components/PendingMemberActions/index.tsx

```diff
@@ -21,7 +21,7 @@ export const PendingMemberActions: (props: Props) => JSX.Element = ({
 
 
     return (
-        <span className="u-flex u-justify-between u-w-full">
+        <div className="u-flex u-justify-between u-w-full">
```

Comment:
Could these classes be added directly to the className prop of <Form /> and avoid the need for the wrapping <div />?

Reply:
Think we might be stuck with the wrapping div for this one, it's two forms so I need the flex/justify on a container to spread the buttons rather than the form component itself. I've tried adding the classes to the <td> itself but the styles we're applying in the DataGrid component (i.e table-cell) override the flex/justify on tablet/desktop. 

Not sure if it's worth tinkering with the DataGrid styles for this?